docs/reference/mapping/types.asciidoc

```diff
@@ -56,6 +54,12 @@ string::         <<text,`text`>> and <<keyword,`keyword`>>
 
 <<flattened>>:: Allows an entire JSON object to be indexed as a single field.
 
+[float]
+[[types-array-handling]]
+=== Array handling
```

Comment:
Super small comment, maybe `Arrays` would read more naturally, especially given the `Multi-fields` title below?

Reply:
Thanks. I agree this would be more consistent.

Changed with e8f6d429ec2b43a4a9e0b39177a88865576f9579.